src/sessions/__tests__/PageManager.test.ts

```diff
@@ -487,3 +488,86 @@ describe('PageManager tests', () => {
         expect(pageManager.getPage().start).toEqual(2500);
     });
 });
+
+test('when complete referrer is available from the DOM', async () => {
```

Comment:
suggestion: `when complete referrer is available from the DOM` --> `when complete referrer is available from the DOM then is recorded in page view event`